langchain/src/document_loaders/base.ts

```diff
@@ -10,7 +10,7 @@ export interface DocumentLoader {
 }
 
 export abstract class BaseDocumentLoader implements DocumentLoader {
-  abstract load(): Promise<Document[]>;
+  abstract load(selector?: unknown): Promise<Document[]>;
```

Comment:
Thanks for this, I think the selector should be moved to a constructor argument on CheerioWebBaseLoader

Reply:
I do think so, its weird to add a parameter 'selector' in the abstract BaseDocumentLoader, especially you do implementation in other tools. Its not an universal parameter